bundles/org.openhab.binding.matter/src/main/java/org/openhab/binding/matter/internal/client/dto/cluster/gen/SemanticTags.java

```diff
@@ -0,0 +1,705 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+// AUTO-GENERATED, DO NOT EDIT!
+
+package org.openhab.binding.matter.internal.client.dto.cluster.gen;
+
+/**
+ * Matter Semantic-Tag definitions.
+ *
+ * @author Dan Cunningham - Initial contribution
+ */
+public class SemanticTags {
```

Comment:
Would be nice to have an extra pair of eyes @andrewfg are you available?

Reply:
Thanks, i was a bit to quick recognizing these as the openHAB SemanticTags while they are not. Sorry.